src/containers/Signup/style.css

```diff
@@ -0,0 +1,100 @@
+.login__logo{
+    margin-top: 20px;
+    margin-bottom: 20px;
+    object-fit: contain;
+    width: 100px;
+    margin-right: auto;
+    margin-left:auto;
+}
+.heading{
+    font-family: "Amazon Ember" Arial sans-serif;
```

Comment:
if possible provide this font-family to body so that you dont have to write on each place.